service/camera-streamer-pi219-8MP.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=camera-streamer web camera
+After=network.target
+
+[Service]
+ExecStart=/usr/local/bin/camera-streamer \
```

Comment:
I was also testing a change to run this service only if camera is present.

Can you add and validate this line before `ExecStart`:

```bash
ConditionPathExists=/sys/bus/i2c/drivers/imx219/10-0010/video4linux
```

Reply:
Added and rebooted my Pi to ensure it still started